annlite/executor.py

```diff
@@ -111,6 +150,11 @@ def update(
 
             - 'traversal_paths' (str): traversal path for the docs
         """
+        if len(self._task_queue) > 0:
+            self.logger.info(
+                'updateing operation is not allowed when len(task queue) > 0'
+            )
+            return
```

Comment:
no, raise an exception to notify the user the `update` request is not processed.